host_v2/src/containers/GameSessionContainer.tsx

```diff
@@ -128,42 +138,70 @@ export default function GameSessionContainer({
   const currentTimer = 90;
 
   const totalTime =
-    currentState === GameSessionState.CHOOSE_CORRECT_ANSWER
+    gameSession.currentState === GameSessionState.CHOOSE_CORRECT_ANSWER
       ? phaseOneTime
       : phaseTwoTime;
 
-  const handleTimerIsFinished = () => {
-    console.log('timer is finished'); // eslint-disable-line
-  };
 
-  return (
-    <StackContainerStyled>
-      <HeaderBackgroundStyled />
-      <HeaderContent
-        currentState={currentState}
-        totalQuestions={totalQuestions}
-        currentQuestionIndex={currentQuestionIndex}
-        statePosition={statePosition}
-        isCorrect={isCorrect}
-        isIncorrect={isIncorrect}
-        totalTime={totalTime}
-        currentTimer={hasRejoined ? currentTimer : totalTime}
-        isPaused={false}
-        isFinished={false}
-        handleTimerIsFinished={handleTimerIsFinished}
-        localModel={localModelMock}
-      />
-      <BodyStackContainerStyled>
-        <BodyBoxUpperStyled />
-        <BodyBoxLowerStyled />
-        <PlaceholderContentArea
-          confidenceData={sampleConfidenceData}
-          confidenceGraphClickIndex={confidenceGraphClickIndex}
-          handleConfidenceGraphClick={handleConfidenceGraphClick}
+
+  const shortAnswerResponse = [
+    {
+      rawAnswer: 'y=mx+b',
+      normAnswer: 'y=mx+b',
+      isCorrect: true, // only every one
+      isSelectMistaked: true, 
+      count: 2,
+      teams: ['Name1', 'Name2']
+    },
+    {
+      rawAnswer: 'y=mx+4',
+      normAnswer: 'y=mx+4',
+      isCorrect: false,
+      isSelectMistaked: true, 
+      count: 1,
+      teams: ['Name3']
+    },
+    {
+      rawAnswer: 'y=mx+4',
+      normAnswer: 'y=mx+4',
+      isCorrect: false,
+      isSelectMistaked: true, 
+      count: 5,
+      teams: ['Name4', 'Name5', 'Name6', 'Name7']
+    }
+  ];  
+
```

Comment:
this switch statement replaces the routing we were doing in `App` previously

Reply:
If you want to test StartGame, you can edit the mock data instead of navigating via URL. This follows more closely to what the actual app will do